ex6_ch2.py

```diff
@@ -0,0 +1,25 @@
+#!/usr/bin/env python3
+# -*- coding: utf-8 -*-
+"""
+Created on Thu May 10 15:34:22 2018
+
+@author: amyhaddad
+"""
+
+#step1: ask user for input: age
+current_age = int(input("What is your age? "))
+
+#step2: ask user for input: ideal retirement age
+ideal_retirement_age = int(input("What age do you want to retire? "))
+
+#step3: write calculation to determine the number of years until retirement
+years_to_retirement = ideal_retirement_age - current_age
+
+#step4: write calculation to determine the year you can retire
+retirement_year = 2018 + years_to_retirement
+
```

Comment:
As a challenge, the book says not to hardcode the year since the computer knows what year it is. So if I don't type "2018," do I have to insert the following to get the program to insert the year:

import datetime
currentDT = datetime.datetime.now()
print ("Current Year is: %d" % currentDT.year)

Is there another way to do this? Do I just need to memorize this?

Reply:
Yes, you'll usually have to look it up. Even I do for this function. The bigger thing is to understand that there are DateTimes in Python.